rclcpp/include/rclcpp/experimental/intra_process_manager.hpp

```diff
@@ -203,22 +205,18 @@ class IntraProcessManager
       // None of the buffers require ownership, so we promote the pointer
       std::shared_ptr<MessageT> msg = std::move(message);
 
-      this->template add_shared_msg_to_buffers<MessageT, Alloc, Deleter>(
+      this->template add_shared_msg_to_buffers<MessageT, Alloc, Deleter, ROSMessageType>(
         msg, sub_ids.take_shared_subscriptions);
     } else if (!sub_ids.take_ownership_subscriptions.empty() && // NOLINT
       sub_ids.take_shared_subscriptions.size() <= 1)
     {
-      // There is at maximum 1 buffer that does not require ownership.
```

Comment:
why removing this comment? I think it still applies

Reply:
This should be fixed in the latest commits.